src/main/java/org/junit/ClassRule.java

```diff
@@ -27,8 +27,11 @@
  * If there are multiple
  * annotated {@link ClassRule}s on a class, they will be applied in an order
  * that depends on your JVM's implementation of the reflection API, which is
- * undefined, in general. However, Rules defined by fields will always be applied
- * before Rules defined by methods.
+ * undefined, in general. However, Rules defined by methods will always be applied
+ * before Rules defined by fields.
+ * Since almost all of the in-box {@link org.junit.rules.TestRule} implementations are adding
```

Comment:
Yes, I disagree. Let's drop this sentence and take Marc Philipp's 
> 
Rules defined by fields will always be applied after Rules defined by methods, i.e. the Statements returned by the former will be executed around those returned by the latter.